### PR TITLE
#131 Implemented AwsDocs.size()

### PR DESCRIPTION
**Pull request by carlosmiranda**

### CODE REVIEW

**Comment by karato** [comment]
Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@darkled review it

File: src/test/java/com/nerodesk/om/aws/AwsDocsTest.java [hunk 1]
```diff
@@ -40,6 +48,34 @@
...
+     *  this test. However, the ObjectMetadata that is returned by the mocking
+     *  framework does not show an accurate contentLength. This is the reason
+     *  we're using Mockito. When jcabi-s3 supports this feature (see
+     *  https://github.com/jcabi/jcabi-s3/issues/25), let's migrate this test
+     *  to com.jcabi.s3.mock classes.
+     */
+    @Test
+    public void fetchesSize() throws Exception {
+        final long size = Tv.THOUSAND;
+        final Bucket bucket = Mockito.mock(Bucket.class);
```
Comment (asaen) [inline src/test/java/com/nerodesk/om/aws/AwsDocsTest.java:65]: could we use `com.jcabi.s3.mock.MkBucket` instead?

File: src/test/java/com/nerodesk/om/aws/AwsDocsTest.java [hunk 2]
```diff
@@ -40,6 +48,34 @@
...
+        final Ocket ocket = Mockito.mock(Ocket.class);
```
Comment (asaen) [inline src/test/java/com/nerodesk/om/aws/AwsDocsTest.java:66]: could we use `com.jcabi.s3.mock.MkOcket` instead?

**Comment by asaen** [comment]
@carlosmiranda few comments above

File: src/test/java/com/nerodesk/om/aws/AwsDocsTest.java (hunk 1 above)
Comment (carlosmiranda) [inline src/test/java/com/nerodesk/om/aws/AwsDocsTest.java:65]: No - see the puzzle.

File: src/test/java/com/nerodesk/om/aws/AwsDocsTest.java (hunk 2 above)
Comment (carlosmiranda) [inline src/test/java/com/nerodesk/om/aws/AwsDocsTest.java:66]: We can't (with current jcabi-s3 version). See the puzzle that I created, issue has been raised in jcabi-s3.

**Comment by carlosmiranda** [comment]
@darkled Yes I considered using jcabi-s3 mock classes. They don't work for this scenario because they lack metadata. That's why I put the puzzle in.

**Comment by asaen** [comment]
@carlosmiranda yes, you're right, sorry.

**Comment by asaen** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by karato** [comment]
@darkled Many thanks! **18 mins** were added to your account in Transaction ID `54254764` (task took 17 hours and 35 mins)

review comments (c=3) added as a bonus

+18 added to your rating, current score is: [+2278](http://www.netbout.com/b/35091?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**